metadata-ingestion/tests/integration/delta_lake/golden_files/local/golden_mces_tables_with_nested_datatypes.json

```diff
@@ -1796,397 +1796,6 @@
         "lastRunId": "no-run-id-provided"
     }
 },
-{
-    "entityType": "dataset",
-    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:delta-lake,my-platform.tests/integration/delta_lake/test_data/delta_tables_nested_datatype/table_with_string_and_nested_array_of_numbers,UAT)",
-    "changeType": "UPSERT",
-    "aspectName": "browsePathsV2",
```

Comment:
Note for reviewers:
No idea why these `browsePathsV2` aspects are gone, any clue?

Reply:
Definitely, this is related to https://github.com/datahub-project/datahub/blob/21b3c4773f3aac54767cd5291d6d249bc88a1346/metadata-ingestion/src/datahub/utilities/hive_schema_to_avro.py#L31

As soon as I revert that, `browsePathsV2` aspects are back. So this requires some investigation

Reply:
What I have found is that there are no missing `browsePathsV2` aspects actually. They were duplicated in the golden file and not anymore!